.github/workflows/build_and_test.yml

```diff
@@ -27,8 +27,9 @@ jobs:
           access_token: ${{ secrets.GITHUB_TOKEN }}
       
   build-test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Fixed the version, since we build specifically for Ubuntu 20.04 atm, an automatic update of 'latest' to `22.04` would break our build.